plugins/janus_videoroom.c

```diff
@@ -356,13 +356,15 @@ room-<unique room ID>: {
 }
 \endverbatim
  *
- * To get a list of the available rooms (excluded those configured or
- * created as private rooms) you can make use of the \c list request,
- * which has to be formatted as follows:
+ * To get a list of the available rooms you can make use of the \c list request.
+ *
```

Comment:
Please remove this extra line, or doxygen will create a new paragraph. Since the text below is a continuation/expansion to the intro to `list` itself, it makes sense to keep in contiguous.

Reply:
done